### ISSUE TITLE
featured image to tweets linked to wp post url

### ISSUE THREAD

**Issue by a66fm** [10up/autoshare-for-twitter#244]
### Is your enhancement related to a problem? Please describe.

instead of opening the image in a pop up windows i would like to have the featured image to tweets linked to open the wp post url

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jeffpaul** [comment]
@a66fm I'm not sure I follow your request, could you please restate how things are currently working and how you'd like that to change?

**Comment by a66fm** [comment]
currently the tweet consists of part of the title since the character limit is also adding up the url characters number and the featured image which will open up when clicked
it would be grate to have tweeted either the title with the"twiter card" or the title with the featured image linked to the wp article so when the photo is clicked the article is opened instead of a zoomed version of the featured image
thanks in advance

**Comment by justinmaurerdotdev** [comment]
Was just looking at these "help wanted" issues and this one stands out to me as probably unnecessary. We initially had the same issue, but figured out the cause. The "card" view is rendered by Twitter based on meta values set in the HTML of the shared page. By default, [Twitter renders the image set in the twitter:image](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image) (possibly falling back to og:image) meta tags. This just is the behavior when a link to a properly-tagged URL is shared _and no image is included_.

So, basically, users can have the card view rendered by _not_ publishing the tweet with the image. The tweet will still contain the desired text, but will also include the article preview.  Simply uncheck the "Always add the featured image to tweets" box in the Autoshare settings, and then @a66fm will have the desired functionality.

Maybe I'm missing something?

**Comment by jeffpaul** [comment]
@justinmaurerdotdev perhaps something to add as an FAQ in the readme?

**Comment by lekosbelas** [comment]
So are you saying it's not possible? I tried the both option checked and unchecked. I want the middle option, I want the full image that takes you to your website. I can see that many pp have/had.

**Comment by jeffpaul** [comment]
@iamdharmesh what's your take on best approach here?

**Comment by iamdharmesh** [comment]
@jeffpaul I am agree with @justinmaurerdotdev. Simply uncheck the "Always add the featured image to tweets" box in the Autoshare settings and make sure that your markup has proper needed meta tags.

**Comment by jeffpaul** [comment]
@iamdharmesh perhaps something we document in an FAQ in the readme?
